.github/workflows/test-php.yml

```diff
@@ -12,24 +12,42 @@ on:
 jobs:
   test-php:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['8.1']
+        composer-mode: ['low-deps', 'high-deps']
 
     steps:
       - uses: actions/checkout@v3
 
-      - name: Cache Composer dependencies
-        uses: actions/cache@v3
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
```

Comment:
I don't know if this is 'better' than the other action but is sort of the default one OSS projects I'm involved with use